pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -4498,7 +4501,7 @@ def CompletionsWaterDeliveriesRule(model, p, t):
     return model
 
 
-def water_quality(model, df_parameters, df_sets):
+def water_quality(model):
     # region Fix solved Strategic Model variables
     for var in model.component_objects(Var):
         for index in var:
```

Comment:
We are still rounding small values to 0 when fixing variables prior to solving water quality. I quickly tested removing this rounding and the model is infeasible -- this is something I plan to address in a future PR.